src/colmap/controllers/incremental_pipeline.h

```diff
@@ -135,9 +135,9 @@ struct IncrementalPipelineOptions {
   std::string snapshot_path = "";
   int snapshot_images_freq = 0;
 
-  // Which images to reconstruct. If no images are specified, all images will
-  // be reconstructed by default.
-  std::unordered_set<std::string> image_names;
+  // Optional list of image names to reconstruct. If no images are specified,
+  // all images will be reconstructed by default.
+  std::vector<std::string> image_list;
```

Comment:
I think that `image_names` is more informative but no strong preference.

Reply:
No opinion from my side other than making it consistent. Changed it to image_names.